bip-0158.mediawiki

```diff
@@ -323,7 +323,8 @@ though it requires implementation of the new filters.
 
 We would like to thank bfd (from the bitcoin-dev mailing list) for bringing the
 basis of this BIP to our attention, Greg Maxwell for pointing us in the
-direction of Golomb-Rice coding and fast range optimization, and Pedro
+direction of Golomb-Rice coding and fast range optimization, Pieter Wullie for
+his analysis of optimal gcs parameters, and Pedro
```

Comment:
nit: s/gcs/GCS/ for consistency with the rest of the document.

Reply:
Fixed in all areas other than the pseudocode.